pkg/dfmt.pkg

```diff
@@ -0,0 +1,28 @@
+# dfmt packaging configuration
+# ============================
+
+dfmt_bindir='bin'
+
+#-------------------------------------------------------------------------------
+#    Define the keyword arguments to pass to fpm (via the OPTS dict)
+#-------------------------------------------------------------------------------
+
+OPTS.update(
+    name = "dfmt",
+    url = 'https://github.com/dlang-community/dfmt',
+    maintainer = 'Stefan Koch <stefan.koch@sociomantic.com>',
+    vendor = 'Sociomantic Labs GmbH',
+    provides = "dfmt",
+    description = '''\
+D source code formatter
+''',
+    depends = FUN.autodeps('dfmt', path=dfmt_bindir),
+)
+
+#-------------------------------------------------------------------------------
+#    Define the positional arguments to pass to fpm (via the ARGS list)
+#-------------------------------------------------------------------------------
+
+ARGS.extend(FUN.mapfiles(dfmt_bindir, '/usr/bin', 'dfmt'))
```

Comment:
Maybe add the README too?

Reply:
I don't need the readme in the target file-system.
but the bash-completion

Reply:
Is nice to provide a README for users that install the package. Is the standard practice for debian packages, they usually go to `/usr/share/doc/<pkg>/`.